features/admin.server-configurations.v1/forms/password-recovery-form/password-recovery-form.tsx

```diff
@@ -0,0 +1,802 @@
+                    { t("extensions:manage.serverConfigurations.accountRecovery." +
+                            "passwordRecovery.form.fields.maxFailedAttemptCount.hint") as ReactNode }
+                </Hint>
+                <Field.Input
+                    ariaLabel="maxResendCount"
+                    inputType="number"
+                    name="maxResendCount"
+                    min={
+                        GovernanceConnectorConstants.PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS
+                            .MAX_RESEND_COUNT_MIN_VALUE
+                    }
+                    max={
+                        GovernanceConnectorConstants.PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS
+                            .MAX_RESEND_COUNT_MAX_VALUE
+                    }
+                    label={ t("extensions:manage.serverConfigurations.accountRecovery." +
+                        "passwordRecovery.form.fields.maxResendCount.label") }
+                    placeholder={ t("extensions:manage.serverConfigurations.accountRecovery." +
+                        "passwordRecovery.form.fields.maxResendCount.placeholder") }
+                    required={ false }
+                    maxLength={
+                        GovernanceConnectorConstants
+                            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.MAX_RESEND_COUNT_MAX_LENGTH
+                    }
+                    minLength={
+                        GovernanceConnectorConstants
+                            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.MAX_RESEND_COUNT_MIN_LENGTH
+                    }
+                    readOnly={ readOnly }
+                    width={ 10 }
+                    labelPosition="right"
+                    disabled={ !isConnectorEnabled }
+                    data-testid={ `${ testId }-otp-resend-count` }
+                >
+                    <input/>
+                    <Label
+                        content={ "attempts" }
+                    />
+                </Field.Input>
+                <Hint>
+                    { t("extensions:manage.serverConfigurations.accountRecovery." +
+                            "passwordRecovery.form.fields.maxResendCount.hint") as ReactNode }
+                </Hint>
+                <Field.Button
+                    form={ FORM_ID }
+                    size="small"
+                    buttonType="primary_btn"
+                    ariaLabel="Password Recovery update button"
+                    name="update-button"
+                    data-testid={ `${ testId }-submit-button` }
+                    disabled={ isSubmitting }
+                    loading={ isSubmitting }
+                    label={ t("common:update") }
+                    hidden={ !isConnectorEnabled || readOnly }
+                />
+            </Form>
+        </div>
+    );
+};
+
+/**
+ * Default props for the component.
+ */
+PasswordRecoveryConfigurationForm.defaultProps = {
+    "data-testid": "password-recovery-edit-form"
```

Comment:
refrain from using `data-testid` in new components

Reply:
This is not a new component. The file is being detected as new due to the file being moved (to add scss). Changing this might cause test suite failures.